css/popup.css

```diff
@@ -0,0 +1,60 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Press Start 2P', cursive;
+}
+
+.restart {
+    visibility: hidden;
+    width: 800px;
+    height: 300px;
+    position: absolute;
+    padding: 30px;
+    left: 100px;
+    top: 100px;
+    border-radius: 5px;
+    background-color: #b3ff0f;
+}
+
+.restart h2  {
+    text-align: center;
+    font-size: 2rem;
+}
+
+.restart h1  {
```

Comment:
Two spaces after h2, one is used in above selectors.

Reply:
done